eb_ssm/eb_ssm.py

```diff
@@ -48,8 +53,18 @@ def _parse_args(self):
             default=None,
             help="use a specific region",
         )
+        parser.add_argument(
+            "-c", "--command",
+            default=None,
+            help="command to execute",
+        )
+        parser.add_argument(
```

Comment:
Why does it need a non-interactive flag?  If you pass a command that takes no input, it should just terminate upon completion.

If the goal is to avoid the instance selection prompt, this flag should be named something else.  I'd prefer mimicking the eb-ssh flag "--number", which specifies which instance number to ssh into.

Reply:
done